odoo/addons/base/tests/test_ormcache.py

```diff
@@ -12,12 +12,12 @@ def test_ormcache(self):
         XMLID = 'base.group_no_one'
 
         # retrieve the cache, its key and stat counter
-        cache, key, counter = get_cache_key_counter(IMD.xmlid_lookup, XMLID)
+        cache, key, counter = get_cache_key_counter(IMD._xmlid_lookup, XMLID)
         hit = counter.hit
         miss = counter.miss
 
-        # clear the cache of ir.model.data.xmlid_lookup, retrieve its key and
-        IMD.xmlid_lookup.clear_cache(IMD)
+        # clear the cache of ir.model.data._xmlid_lookup, retrieve its key and
+        IMD._xmlid_lookup.clear_cache(IMD)
```

Comment:
```suggestion
        # clear the cache of ir.model.data._xmlid_lookup, retrieve its key and
        IMD.clear_caches()
```

Targeted cache validation isn't supported anymore, the whole cache is cleared anyway (Cf #62800)